src/script/main/app.ts

```diff
@@ -472,7 +472,7 @@ export class App {
 
       await conversationRepository.init1To1Conversations(connections, conversations);
 
-      if (supportsMLS()) {
+      if (this.core.hasMLSDevice) {
```

Comment:
this one is the fix to automation correct?
Seems like the change in `MLSConversationVerificationStateHandler` isn't necessarry? (that's the one breaking unit tests)

Reply:
exactly, I'm working on the fix right now :)